docs/niscope/class.rst

```diff
@@ -1460,6 +1458,94 @@ fetch
 
 
 
+fetch_array_measurement
+-----------------------
+
+    .. py:currentmodule:: niscope.Session
+
+    .. py:method:: fetch_array_measurement(array_meas_function, meas_wfm_size, timeout=hightime.timedelta(seconds=5.0), other_channel=None)
+
+            Obtains a waveform from the digitizer and returns the specified
+            measurement array. This method may return multiple waveforms depending
+            on the number of channels, the acquisition type, and the number of
+            records you specify.
+
+            
+
+            .. note:: Some functionality, such as time stamping, is not supported in all
+                digitizers.
+
+
+            .. tip:: This method requires repeated capabilities. If called directly on the
+                niscope.Session object, then the method will use all repeated capabilities in the session.
+                You can specify a subset of repeated capabilities using the Python index notation on an
+                niscope.Session repeated capabilities container, and calling this method on the result.
+
+
+            :param array_meas_function:
+
+
+                The array measurement to perform.
+
+                
+
+
+            :type array_meas_function: :py:data:`niscope.ArrayMeasurement`
+            :param timeout:
+
+
+                The time to wait in seconds for data to be acquired; using 0 for this
+                parameter tells NI-SCOPE to fetch whatever is currently available. Using
+                -1 for this parameter implies infinite timeout.
+
+                
+
+
+            :type timeout: hightime.timedelta, datetime.timedelta, or float in seconds
+            :param other_channel:
+
+
+                The identifier for the "other channel" for multi-channel measurements such as Add Channels or Multiply Channels.
+
+                
+
+
+            :type other_channel: str
+
+            :rtype: list of WaveformInfo
+            :return:
+
+
+                    Returns a list of class instances with the following timing and scaling
+                    information about each waveform:
+
+                    -  **relativeInitialX**—the time (in seconds) from the trigger to the
+                       first sample in the fetched waveform
+                    -  **absoluteInitialX**—timestamp (in seconds) of the first fetched
+                       sample. This timestamp is comparable between records and
+                       acquisitions; devices that do not support this parameter use 0 for
+                       this output.
+                    -  **xIncrement**—the time between points in the acquired waveform in
+                       seconds
+                    -  **channel**-channel name this waveform was acquired from
+                    -  **record**-record number of this waveform
+                    -  **samples**—floating point array of samples. Length will be of actual samples acquired
```

Comment:
Nitpick: In other places where we return this type, `samples` is documented at the end. Any particular reason why not be consistent with them?

Reply:
This probably wasn't an intentional ordering decision. I'll move it around.